scripts/install-exec-dependencies.sh

```diff
@@ -21,11 +21,16 @@ get_dns_suffix() {
         exit 1
     fi
```

Comment:
just to confirm, aws-eusc partition is not considered air-gapped?

Very minor nit to just update the comment on this function.